charts/all/medical-diagnosis/grafana/templates/xraylab-dashboard.yaml

```diff
@@ -363,7 +363,7 @@ spec:
           "timeShift": null,
           "title": "Last uploaded image",
           "type": "ryantxu-ajax-panel",
-          "url": "https://image-server-xraylab-1.apps.{{ .Values.global.datacenter.clustername }}.{{ .Values.global.datacenter.domain }}/last_image_small/{{ .Values.global.xraylab.s3.bucketBaseName }}",
+          "url": "https://image-server-xraylab-1.apps.{{ .Values.global.clusterDomain }}/last_image_small/{{ .Values.global.xraylab.s3.bucketBaseName }}",
```

Comment:
@day0hero why the non-standard DNS variables?  shouldn't this be coming from the operator's automatic variables?